docs/installation.md

```diff
@@ -30,6 +30,14 @@ In Github releases you will find a `clojure-lsp` file that runs a embedded jar.
 nix-shell -p clojure-lsp
 ```
 
+## Arch Linux
+
+`clojure-lsp` is
+[available](https://aur.archlinux.org/packages/clojure-lsp-bin/) in the [Arch User Repository](https://aur.archlinux.org). It can be installed using your favorite [AUR](https://aur.archlinux.org) helper such as
+[yay](https://github.com/Jguer/yay), [yaourt](https://github.com/archlinuxfr/yaourt), [apacman](https://github.com/oshazard/apacman) and [pacaur](https://github.com/rmarquis/pacaur). Here is an example using `yay`:
+
+    yay -S clojure-lsp-bin
```

Comment:
Just curious, why this is not just `clojure-lsp`?

Reply:
Hi. Arch naming convention. Anything which is "just" a bin(ary) file, is named as a `-bin` package. However, that's just the name of the package. The actual executable is still installed as `clojure-lsp`

Reply:
another example `https://aur.archlinux.org/packages/clj-kondo-bin/`

Reply:
Good to know, thanks for the explanation